<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_9">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QLabel" name="pubkey_left_label">
                <property name="text">
                 <string>公開鍵</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="pubkey_left_lineEdit"/>
              </item>
             </layout>
            </item>
            <item row="0" column="1">
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QLabel" name="privkey_left_label">
                <property name="text">
                 <string>秘密鍵</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="privkey_left_lineEdit"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="asymkey_pushButton">
            <property name="text">
             <string>生成</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pubkey_send_pushButton">
          <property name="text">
           <string>--送信--&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>公開鍵</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_5"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_13">
            <item>
             <widget class="QLabel" name="dec_symkey_left_label">
              <property name="text">
               <string>暗号化共通鍵</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="dec_symkey_left_lineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="dec_symkey_pushButton">
            <property name="text">
             <string>復号化</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_14">
            <item>
             <widget class="QLabel" name="symkey_left_label">
              <property name="text">
               <string>共通鍵</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="symkey_left_lineEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="symkey_send_pushButton">
          <property name="text">
           <string>&lt;--送信--</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="symkey_right_label">
              <property name="text">
               <string>共通鍵</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="symkey_gen_pushButton">
              <property name="text">
               <string>生成</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QLineEdit" name="symkey_right_lineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="enc_symkey_pushButton">
            <property name="text">
             <string>暗号化</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QLabel" name="enc_symkey_right_label">
              <property name="text">
               <string>暗号化共通鍵</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="enc_symkey_right_lineEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="plain_left_label">
              <property name="text">
               <string>平文</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="plain_left_plainTextEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="enc_pushButton">
            <property name="text">
             <string>暗号化</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <widget class="QLabel" name="cipher_left_label">
              <property name="text">
               <string>暗号文</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="cipher_left_plainTextEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="message_send_pushButton">
          <property name="text">
           <string>--送信--&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_15">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QLabel" name="plain_right_label">
              <property name="text">
               <string>平文</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="plain_right_plainTextEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="dec_pushButton">
            <property name="text">
             <string>復号化</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_16">
            <item>
             <widget class="QLabel" name="cipher_right_label">
              <property name="text">
               <string>暗号文</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="cipher_right_plainTextEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
